cmd/sonobuoy/app/gen.go

```diff
@@ -133,6 +136,25 @@ func (g *genFlags) Config() (*client.GenConfig, error) {
 	}, nil
 }
 
+// resolveConformanceImage maps versions before 1.13 to Heptio's image and otherwise
+// to the upstream cnoformance image. Latest is always mapped to the upstream
+// regardless. The comparison is just lexical, e.g. "foo" < "v1.13" and "zip" >
+// "v1.13". These are a-typical and not given more support at this time.
+func resolveConformanceImage(imageVersion string) string {
+	// TODO(johnschnake): This logic should be temporary and is only
```

Comment:
Should make an issue tracking this TODO

Reply:
https://github.com/heptio/sonobuoy/issues/620